WDL/CLI.py

```diff
@@ -434,7 +435,12 @@ async def read_source(uri, path, importer):
                     tmpdir,
                     os.path.basename(parse.urlsplit(uri).path),
                 )
-                request.urlretrieve(uri, filename=fn)
+                headers = {"Authorization": f"token {os.environ.get(GITHUB_PERSONAL_ACCESS_TOKEN_KEY)}"}
```

Comment:
if the env var is not set, maybe it's cleaner to leave out the authorization header alltogether.